composer.json

```diff
@@ -29,7 +29,8 @@
         "symfony/process": "^2.3|^3.0"
     },
     "require-dev": {
-        "psr/log": "^1.0"
+        "psr/log": "^1.0",
+        "phpunit/phpunit": "^4.8"
```

Comment:
I don't think this should be added for two reasons. The first being that the recommendation is to global install phpunit, and the second being that this change is unrelated to the rest of the PR, so should be proposed seperately.